src/components/member-profile/member-profile.module.scss

```diff
@@ -211,6 +211,10 @@
   .memberDetails {
     text-align: center;
   }
+
+  .memberStatus {
+    justify-content: center !important;
```

Comment:
why do we need to put !important here?

Reply:
Actually in desktop it is flex-start so to override it I've put `important` otherwise this property won't work

Reply:
I have removed the !important tag and modified CSS